src/modules/tests/communication-preferences/communication-preferences.selector.ts

```diff
@@ -6,3 +6,6 @@ export const getCommunicationPreferenceUpdatedEmail
 
 export const getCommunicationPreferenceType
   = (communicationPreferences: CommunicationPreferences) => get(communicationPreferences, 'communicationMethod', '');
+
+export const getConductedLanguage
+  = (communicationPreferences: CommunicationPreferences) => get(communicationPreferences, 'conductedLanguage', '');
```

Comment:
Do we really need a `get` here? Is there a case we need to guard against null? If there is, for 1 level deep I might be inclined to do it manually.

Reply:
Done